documentation/src/main/asciidoc/core/manual/en_US/05_predicate_builder.adoc

```diff
@@ -20,7 +20,7 @@ There are multiple different entry methods to cover all possible usage scenarios
 e.g. in the `WHERE` clause the entry methods are named `where()`, `whereExists()` etc.
 The following list of possible entry methods refers to `WHERE` clause entry methods for easier readability.
 
-link:{core_jdoc}/persistence/WhereBuilder.html#whereExpression(java.lang.String)[`where(String expression)`]::
+link:{core_jdoc}/persistence/WhereBuilder.html#where(java.lang.String)[`where(String expression)`]::
```

Comment:
Shouldn't this be setWhereExpression?

Reply:
Nope, that's on purpose. This is the entry point for starting with the predicate builder API. I just wrongly referenced a different method there before